### DISCUSSION TITLE
Screen flickering on profile tab

### DISCUSSION THREAD

**Discussion by JonathanBout** [community/community#124145] [Mobile] [answered]
### Select Topic Area

Bug

### Body

On the profile tab the readme is rapidly expanding and closing as visible here:
https://github.com/community/community/assets/93737747/f49824fa-9e15-48d5-bcdc-69ef969cebe4

A force stop and clearing the cache of the app didn't solve it.

**Comment by aguluman** [comment]
> ### Select Topic Area
> 
> Bug
> 
> ### Body
> 
> On the profile tab the readme is rapidly expanding and closing as visible here:
> https://github.com/community/community/assets/93737747/f49824fa-9e15-48d5-bcdc-69ef969cebe4
> 
> A force stop and clearing the cache of the app didn't solve it.

On my profile tab, its showing try again.
I have restarted the application, it still shows the same thing.

**Comment by alcere** [comment, marked as answer]
@JonathanBout We apologize for the inconvenience! We have just released a Beta update that should resolve the issue you mentioned. Would you mind giving it a try, please?

**Reply by JonathanBout** [reply]
Yep, it works again, thank you!
